core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1034,16 +1035,21 @@ private[group] class GroupCoordinator(
       if (validationErrorOpt.isDefined) {
         responseCallback(offsetMetadata.map { case (k, _) => k -> validationErrorOpt.get })
       } else {
+        // GroupMetadataManager does not support topic ids yet.
```

Comment:
I don't really see the need for `TopicIdAdaptor`. Could we just update `storeOffsets` to take `TopicIdPartition` as well? Updating the method signatures is fine but we should not update the underlying Map where offsets are stored. Am I missing something?

Reply:
You are right, it is simpler to propagate one more layer until the final data is constructed. Thanks, updating the PR.